sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
+have only recently found widespread use [@tag:Schmidhuber2014_dnn_overview].
+For biologists, such problems are further complicated by the immense size of
+most biological datasets.
+
+*TODO: Perhaps a visual illustrating the changes over time in the cost of
+storage
+(i.e. hard drive space), genome sequencing, and processing power. Additionally,
```

Comment:
* I agree, the GPU cores and RAM would definitely be good to have. I will see if I can find a review that includes such information.
* I have already compiled a list of ImageNET classification task winners with accuracy, along with citations for the models where available. However, I am still unsure about how to quantify the increasing complexity of the winning networks, as there is clearly a distinction to be made between types of layers, and I don't know how useful parameter counts would be.
* The data in [@tag:Stein2010_cloud] is missing recent years. This is not really a problem, and I can find more data regarding storage costs if that ends up being something we want to have.